code/modules/mob/living/carbon/human/human_species.dm

```diff
@@ -10,7 +10,7 @@
 	delete_inventory()
 
 /mob/living/carbon/human/skrell/New(var/new_loc)
-	h_style = "Skrell Male Tentacles"
+	h_style = "Skrell Short Tentacles"
```

Comment:
Wouldn't the average ones make more sense?

Reply:
Doesn't really matter.